src/components/pdf/Certificate.tsx

```diff
@@ -14,7 +14,15 @@ Font.load({ fontFamily: 'Roboto' })
 
 const styles = StyleSheet.create({
   page: {
-    backgroundColor: '#E1E4E5',
+    backgroundColor: '#fff',
+  },
+  backgroundImage: {
+    position: 'absolute',
+    height: '100%',
+    width: '100%',
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
```

Comment:
What's the error we ignore here?

Maybe it's better to use `@ts-expect-error` instead of `@ts-ignore`